packages/www/pages/contact.tsx

```diff
@@ -150,187 +133,16 @@ const ContactPage = () => {
               id="subject"
               name="TICKET.subject"
               type="text"
-              placeholder="Ticket subject*"
+              placeholder="What's the topic?*"
```

Comment:
Thoughts on removing this field? Is it necessary if we have the message field below it?

Reply:
The original thought of it was to help filter out the inquiry in relation to sales support, product feature, or tech support.

Reply:
Gotcha. Since it's an open ended text field and we don't explicitly allow users to choose between those topics should we remove now?